dts/arm/microchip/mec1501hsz.dtsi

```diff
@@ -36,6 +36,12 @@
 			interrupts = <111 0>;
 			label = "RTIMER";
 		};
+		wdog: watchdog@40000400 {
```

Comment:
Is there a reason this is not called wdt, which is the short name used everywhere else

Reply:
Actually you found an inconstancy inside dts: some use wdog (atmel and nxp basically), some use wdt (nrf ones only). Hard to tell which is the one to use. I won't touch it atm, but it might be worth to resolve this dts wording at some point.

Reply:
Alright